tests/common/dualtor/dual_tor_utils.py

```diff
@@ -441,8 +446,16 @@ def shutdown_fanout_upper_tor_intfs(upper_tor_host, upper_tor_fanouthosts, tbinf
     shut_fanouts = []
     fanout_intfs_to_recover.clear()
 
+    mux_ports = []
+    if cable_type == CableType.active_active:
+        mux_ports = active_active_ports
+    elif cable_type == CableType.active_standby:
+        mux_ports = active_standby_ports
```

Comment:
This chunk of code repeated 3 times, maybe it should be a helper function?

Reply:
Maybe use code like blow:
```
mux_ports = active_active_ports if cable_type == CableType.active_active else active_standby_ports
```
Not exactly the same logic but it should work.